hoomd/hpmc/UpdaterClusters.h

```diff
@@ -24,6 +24,11 @@
 #include <tbb/parallel_for.h>
 #include <tbb/task.h>
 #include <atomic>
+
+#if TBB_VERSION_MAJOR < 2021
+#define ENABLE_TBB_TASK
+#endif
+
```

Comment:
Why do we check the version here, but not elsewhere?

Reply:
UpdaterClusters uses `tbb::task`, which doesn't have a direct replacement in TBB2021. This code will need a more extensive rewrite to support TBB 2021: #958